tarofreighter/chain_porter.go

```diff
@@ -702,21 +695,6 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
 
 		return &currentPkg, nil
 
-	// Alternatively, we'll enter this state when we know we don't actually
-	// need a split at all. In this case, we fully consume an input asset,
-	// so the asset created is the same asset w/ the new script key in
-	// place.
-	case SendStatePreparedComplete:
```

Comment:
We can remove the enum value definition above as well.

Reply:
Looks like `SendStatePreparedComplete` still exists in `parcel.go`.